src/burnchains/bitcoin/spv.rs

```diff
@@ -332,12 +332,18 @@ impl SpvClient {
 
     /// Report how many block headers we have downloaded to the given path.
```

Comment:
Can you add a comment to explain how `get_headers_height()` differs from `get_highest_header_height()`?

Reply:
addressed with https://github.com/blockstack/stacks-blockchain/pull/2073/commits/854b21e5639c32065e4d1ed150e4df556da62201